data/noiseprofiles.json

```diff
@@ -5051,6 +5051,20 @@
             {"name": "Nexus 5 iso 3200", "iso": 3200, "a": [0.000753506842617197, 0.000424195799109644, 0.000858423239661108], "b": [1.42802529339895e-05, 1.26551608653164e-05, 2.19708111759326e-05]},
             {"name": "Nexus 5 iso 3200", "iso": 3200, "a": [0.000778248212852919, 0.000442218441209912, 0.000922519536168604], "b": [1.15050395187219e-05, 1.01305383004307e-05, 1.47053958885876e-05]}
           ]
+        },
```

Comment:
Please properly rebase this pr, neither noiseprofile nor rawspeed changes should not be here.